apps/api/src/gpu/services/gpu-price.service.ts

```
@@ -0,0 +1,235 @@
+              cpuUnits: decodedBid.resourcesOffer
+                .flatMap(r => (r.resources?.cpu?.units?.val ? parseInt(uint8arrayToString(r.resources.cpu.units.val)) : 0))
+                .reduce((a, b) => a + b, 0),
+              memoryUnits: decodedBid.resourcesOffer
+                .flatMap(r => (r.resources?.memory?.quantity?.val ? parseInt(uint8arrayToString(r.resources.memory.quantity.val)) : 0))
+                .reduce((a, b) => a + b, 0),
+              storageUnits: decodedBid.resourcesOffer
+                .flatMap(r => r.resources?.storage)
+                .map(s => (s?.quantity?.val ? parseInt(uint8arrayToString(s.quantity.val)) : 0))
+                .reduce((a, b) => a + b, 0),
+              gpus: decodedBid.resourcesOffer
+                .filter(x => (x.resources?.gpu?.units?.val ? parseInt(uint8arrayToString(x.resources.gpu.units.val)) : 0 > 0))
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)